fossunited/api/schedule.py

```
@@ -97,7 +121,411 @@ def get_event_schedule(event_id: str) -> dict:
+            lines.append(f":Map_Link: {metadata['Map Link']}")
+        lines += [
+            ":END:",
+            "",
+            "* Description",
+            metadata["Description"],
+            "",
+            "* Schedule",
+        ]
+        for s in sessions:
+            speakers = ", ".join([sp.get("name") for sp in s.speakers_list]) or "N/A"
+            dt_str = f"<{s.scheduled_date} {to_time(s.start_time).strftime('%H:%M')}-{to_time(s.end_time).strftime('%H:%M')}>"  # noqa: E501
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)